ci-operator/step-registry/openshift/e2e/aws/proxy/openshift-e2e-aws-proxy-workflow.yaml

```diff
@@ -0,0 +1,12 @@
+workflow:
+    as: openshift-e2e-aws-proxy
```

Comment:
nit: we seem to prefer two-space indents for step YAML, and this block has four-space indents.